docs/run_tests.html

```diff
@@ -1,14 +1,15 @@
+
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
 <html><head><title>Python: module run_tests</title>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta charset="utf-8">
 </head><body bgcolor="#f0f0f8">
 
 <table width="100%" cellspacing=0 cellpadding=2 border=0 summary="heading">
 <tr bgcolor="#7799ee">
 <td valign=bottom>&nbsp;<br>
 <font color="#ffffff" face="helvetica, arial">&nbsp;<br><big><big><strong>run_tests</strong></big></big></font></td
 ><td align=right valign=bottom
-><font color="#ffffff" face="helvetica, arial"><a href=".">index</a><br><a href="file:/home/semicolonexpected/TaskManager/run_tests.py">/home/semicolonexpected/TaskManager/run_tests.py</a></font></td></tr></table>
+><font color="#ffffff" face="helvetica, arial"><a href=".">index</a><br><a href="file:/Users/alexhong/PycharmProjects/TaskManager/run_tests.py">/Users/alexhong/PycharmProjects/TaskManager/run_tests.py</a></font></td></tr></table>
```

Comment:
actually that probably depends on whos computer this document is on

Reply:
should add docs to gitignore

Reply:
docs are to create an api that other contributers can read to get started.